diff --git a/configure.py b/configure.py
index d85f778..225f647 100644
--- a/configure.py
+++ b/configure.py
@@ -2561,8 +2561,9 @@ def generate_sip_module_code(target_config, verbose, parts, tracing, mname, fata
         argv.append('-f')
 
     # Make sure any unknown Qt version gets treated as the latest Qt v5.
-    argv.append('-B')
-    argv.append('Qt_6_0_0')
+    # argv.append('-B')
+    # argv.append('Qt_6_0_0')
+    # flag is marked as deprecated
 
     if target_config.prot_is_public:
         argv.append('-P');
