diff -urN qtbase-everywhere-src-5.12.7/src/corelib/tools/qsimd_p.h qtbase-everywhere-src-5.12.7-fixed/src/corelib/tools/qsimd_p.h
--- qtbase-everywhere-src-5.12.7/src/corelib/tools/qsimd_p.h	2020-01-23 22:54:31.000000000 -0700
+++ qtbase-everywhere-src-5.12.7-fixed/src/corelib/tools/qsimd_p.h	2020-02-15 15:11:01.525546128 -0700
@@ -346,7 +346,7 @@
 #endif
 Q_CORE_EXPORT void qDetectCpuFeatures();
 
-#if defined(Q_PROCESSOR_X86) && QT_COMPILER_SUPPORTS_HERE(RDRND)
+#if defined(Q_PROCESSOR_X86) && QT_COMPILER_SUPPORTS_HERE(RDRND) && !defined(QT_BOOTSTRAPPED)
 Q_CORE_EXPORT qsizetype qRandomCpu(void *, qsizetype) Q_DECL_NOTHROW;
 #else
 static inline qsizetype qRandomCpu(void *, qsizetype) Q_DECL_NOTHROW
